src/components/Proposal/Create/CreateProposalPage.tsx

```diff
@@ -53,14 +49,17 @@ class CreateProposalPage extends React.Component<IProps, IStateProps> {
     };
   }
 
-  public handleClose = (e: any) => {
-    e.preventDefault();
+  public handleClose = (e?: any) => {
+    if (e?.preventDefault) {
```

Comment:
Why this `if` statement is necessary?

Reply:
Because e is not always exists. When we close window programmaticaly (instead of handle event) we are using this function without param. Any other suggestions?

Reply:
@roienatan does it acceptable from you point of view?

Reply:
It's okay I just wonder since it never caused a problem (at least from what I witnessed) probably it's necessary because of this new feature.

Reply:
@roienatan yes, exactly